src/components/project/dto/update-project.dto.ts

```diff
@@ -25,11 +25,11 @@ export abstract class UpdateProject {
   })
   readonly primaryLocationId?: string;
 
-  @IdField({
-    description: 'A non primary location ID',
+  @Field(() => [ID], {
+    description: 'Other location IDs',
     nullable: true,
   })
-  readonly nonPrimaryLocationId?: string;
+  readonly otherLocationIds?: string[];
```

Comment:
This is different than how we do all other list relationships in the system. I was ok with it for create because there's never any existing relationships, but on update there's existing to think about. i.e. "Is this whole list? and if existing items are missing in this ID list do they need to be deleted?", etc. This is why we have separate mutations to create/delete these list relationships so we know explicitly that this single item needs to be created/deleted instead of having to diff a new list with the existing db state.

The easiest thing to do is just leave this property out of the update mutation right now. We won't implement it immediately on the UI anyways.